libcudacxx/include/nv/detail/__target_macros

```diff
@@ -358,6 +374,14 @@
 #    define _NV_TARGET_BOOL___NV_HAS_FEATURE_SM_90a 0
 #  endif
 
+#  undef NV_HAS_FEATURE_SM_100a
+#  define NV_HAS_FEATURE_SM_100a __NV_HAS_FEATURE_SM_100a
+#  if (defined(__CUDA_ARCH__) && (__CUDA_ARCH__ >= 1000) && defined(__CUDA_ARCH_FEAT_SM100_ALL))
+#    define _NV_TARGET_BOOL___NV_HAS_FEATURE_SM_100a 1
+#  else
+#    define _NV_TARGET_BOOL___NV_HAS_FEATURE_SM_100a 0
+#  endif
+
```

Comment:
I dont think that this is needed yet, as there is no SM100a feature

Reply:
I do think this is needed. SM100a features can build on top of this.

Reply:
I think we possibly don't care. If sm100a doesn't exist then it doesn't matter. The only people that would be bothered are those relying on intellisense.